app/src/main/java/com/nextcloud/talk/controllers/ConversationsListController.java

```diff
@@ -172,23 +184,41 @@ protected void onViewBound(@NonNull View view) {
         prepareViews();
     }
 
+    private void loadUserAvatar(MenuItem menuItem) {
+        if (getActivity() != null) {
+            int avatarSize = (int) DisplayUtils.convertDpToPixel(menuItem.getIcon().getIntrinsicHeight(), getActivity());
+
+            GlideUrl glideUrl = new GlideUrl(ApiUtils.getUrlForAvatarWithNameAndPixels(currentUser.getBaseUrl(),
+                    currentUser.getUserId(), avatarSize), new LazyHeaders.Builder()
+                    .setHeader("Accept", "image/*")
+                    .setHeader("User-Agent", ApiUtils.getUserAgent())
+                    .build());
+
+            GlideApp.with(getActivity())
+                    .asBitmap()
+                    .centerInside()
+                    .override(avatarSize, avatarSize)
+                    .apply(RequestOptions.bitmapTransform(new CircleCrop()))
+                    .load(glideUrl)
+                    .into(new SimpleTarget<Bitmap>() {
+                        @Override
+                        public void onResourceReady(@NonNull Bitmap resource, @Nullable Transition<? super Bitmap> transition) {
```

Comment:
What about onError() and a placeholder while loading and error image as fallback?
Or is this provided via ui / xml?

Reply:
None, but those didn't exist so far either. I might tackle this as part of move to Fresco (there's an issue for this).